splinterd/api/static/openapi.yaml

```diff
@@ -1577,6 +1577,14 @@ paths:
         authenticate the user.
       parameters:
         - $ref: "#/components/parameters/protocol_version"
+        - name: redirect_url
+          in: query
+          description: |-
+            The URL the client will be redirected to once the OAuth
+            authentication process has been completed.
```

Comment:
Should explain what happens when the parameter isn't provided.

Reply:
I specified this in the 400 response description but can add a small explanation here as well. Was kind of awkward finding just the right place to put these descriptions...